src/package.json

```diff
@@ -5,12 +5,12 @@
   "main": "fingerprint-auth",
   "nativescript": {
     "platforms": {
-      "android": "2.5.0",
-      "ios": "2.5.0"
+      "android": "3.0.0",
```

Comment:
It is due to the requirement that the verified plugins should work with the latest version of NativeScript. Since there are breaking changes between v2 and v3, I think the minimum version should be 3.0.0.